### PR TITLE
latest snakemake edits

### PR DESCRIPTION
**Pull request by bintiibrahim**
- fixed gzip bug
- used newer gatk version 4.1.2 in `environment.yaml`
- define analysisDirectory

### CODE REVIEW

**Review by acesnik — DISMISSED** [review]

File: rules/downloads.smk [hunk 1]
```diff
@@ -1,21 +1,24 @@
...
-        "(wget -O - ftp://ftp.ensembl.org/pub/release-81//fasta/homo_sapiens/dna/Homo_sapiens.GRCh38.dna.primary_assembly.fa.gz | "
-        "gunzip -c > {output.gfa} && "
-        "wget -O - ftp://ftp.ensembl.org/pub/release-81/gff3/homo_sapiens/Homo_sapiens.GRCh38.81.gff3.gz | "
-        "gunzip -c > {output.gff} && "
-        "wget -O - ftp://ftp.ensembl.org/pub/release-81//fasta/homo_sapiens/pep/Homo_sapiens.GRCh38.pep.all.fa.gz | "
-        "gunzip -c > {output.pfa} && "
-        "wget -O - ftp://ftp.ncbi.nih.gov/snp/organisms/human_9606_b150_GRCh38p7/VCF/GATK/common_all_20170710.vcf.gz | "
-        "gunzip -c > {output.vcf} && "
-        "gatk IndexFeatureFile -F {output.vcf}) 2> {log}"
+# rule download_ensembl_references:
```
Comment (acesnik) [inline rules/downloads.smk:1]: Why is this commented out?

Comment (bintiibrahim) [reply rules/downloads.smk:1]: the `unzip_ensembl` is built off `download_ensembl_references` so we don't need it anymore

what `unzip_ensembl` does is:
- if the ensembl files exist, unzip
- else, download references

File: environment.yaml [hunk 2]
```diff
@@ -5,7 +5,7 @@ dependencies:
   - snakemake-minimal =5.2.4
   - python =3.6
   - star =2.6.0c
-  - gatk4 =4.0.11.0
+  - gatk4 =4.1.2.0
```
Comment (acesnik) [inline environment.yaml:8]: Was there a reason you updated this?

Comment (acesnik) [reply environment.yaml:8]: Mostly just curious.

**Review by acesnik — APPROVED** [review]

**Pull request merged**